tableauserverclient/server/request_factory.py

```diff
@@ -42,6 +42,7 @@ def _generate_xml(self, datasource_item, connection_credentials=None):
             credentials_element.attrib['name'] = connection_credentials.name
             credentials_element.attrib['password'] = connection_credentials.password
             credentials_element.attrib['embed'] = 'true' if connection_credentials.embed else 'false'
+            credentials_element.attrib['oAuth'] = 'true' if connection_credentials.oauth else 'false'
```

Comment:
I have looked up the [REST API documentation](http://onlinehelp.tableau.com/v10.0/api/rest_api/en-us/help.htm#REST/rest_api_ref.htm#Publish_Datasource%3FTocPath%3DAPI%2520Reference%7C_____43) and only when the connection is oAuth, you should set it to true.

So adding `oAuth="false"` doesn't make any sense. I will update the PR and make the default value set to `False`.